src/v0/gateway.ts

```diff
@@ -1,4 +1,8 @@
 import { ManifoldError, ErrorType } from './ManifoldError';
+import { RestError } from './RestError';
+
+import { Analytics } from './analytics';
+import { waitForAuthToken } from '../utils/auth';
```

Comment:
So this is just a random comment not relating to this PR in particular: if this component is rolled up into a webpack app with code-splitting, every separate file you have (`src/v0/gateway`, `src/v0/analytics`, etc.) will explode and duplicate itself in every chunk. See the PR w/ screenshot here to see what I’m talking about: https://github.com/manifoldco/ui/pull/935

It’s because of how Stencil is set up. If there’s any way at all to load things in the `globalScript` setting, in that entry file, it’s generally best for the clientside. Yeah, the files are messy, but it’s a problem with how Stencil bundles & loads JS internally (which we can solve either by throwing common tools in that one entry file, or we manually configure the Stencil chunks).

Again, random thought, and changing that can happen at any time. But in general: the more files each Stencil repo has, the worse it can be in code-splitting.